client/settings/payment-methods-list/index.js

```diff
@@ -0,0 +1,205 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import {
+	useEnabledPaymentMethodIds,
+	useGetPaymentMethodStatuses,
+	useSelectedPaymentMethod,
+	useUnselectedPaymentMethod,
+} from 'wcpay/data';
+import PAYMENT_METHOD_IDS from 'wcpay/constants/payment-method';
+import PaymentMethod from './payment-method';
+import methodsConfiguration from '../../payment-methods-map';
+import { upeCapabilityStatuses } from 'wcpay/additional-methods-setup/constants';
+import ConfirmPaymentMethodActivationModal from './activation-modal';
+import ConfirmPaymentMethodDeleteModal from './delete-modal';
+import CapabilityRequestNotice from './capability-request';
+import { getMissingCurrenciesTooltipMessage } from 'wcpay/multi-currency/missing-currencies-message';
+
+const PaymentMethodsList = ( { methodIds } ) => {
+	const [ enabledMethodIds ] = useEnabledPaymentMethodIds();
+
+	const paymentMethodStatuses = useGetPaymentMethodStatuses();
+
+	const availableMethods = methodIds.map(
+		( methodId ) => methodsConfiguration[ methodId ]
+	);
+
+	const isCreditCardEnabled = enabledMethodIds.includes(
+		PAYMENT_METHOD_IDS.CARD
+	);
+
+	const [ activationModalParams, handleActivationModalOpen ] = useState(
+		null
+	);
```

Comment:
Formatting nitpick, and this was probably there in the file before, but we could use a newline here.

Reply:
```suggestion
	);

```

Yeah, I think that makes sense - that way they'll be all consistent 👍